### PR TITLE
BZ-1907402: Removing startup probe references as they aren't supporte…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…d in 4.4

https://bugzilla.redhat.com/show_bug.cgi?id=1907402

Preview (internal): http://file.rdu.redhat.com/~ahoffer/2020/BZ-1907402-startup-probes/applications/application-health.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@sttts Can you please take a look?

Preview (internal): http://file.rdu.redhat.com/~ahoffer/2020/BZ-1907402-startup-probes/applications/application-health.html

File: modules/application-health-about.adoc [hunk 1]
```diff
@@ -102,50 +94,6 @@ spec:
...
-  labels:
-    test: health-check
-  name: my-application
-...
-spec:
-  containers:
-  - name: goproxy-app <1>
-    args:
-    image: k8s.gcr.io/goproxy:0.1 <2>
-    livenessProbe: <3>
```
Comment (sttts) [inline modules/application-health-about.adoc:120]: not worth to keep it for liveness probe? or is it only used for the startup probe variant in the text above?

Comment (bergerhoffer) [reply modules/application-health-about.adoc:120]: @sttts One of the other examples does include a liveness probe so I thought that would be sufficient. But if you'd prefer, we could modify the example to remove just the startup probe config. Let me know what you think.

Keep in mind too that this only applies to 4.4, and that all examples will be shown in 4.5+.

Comment (sttts) [reply modules/application-health-about.adoc:120]: am fine with removing it

**Comment by bergerhoffer** [comment]
@sunilcio Can you please review this update? Thanks!

Preview (internal): http://file.rdu.redhat.com/~ahoffer/2020/BZ-1907402-startup-probes/applications/application-health.html

File: modules/application-health-configuring.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@
 [id="application-health-configuring_{context}"]
 = Configuring health checks
 
-To configure readiness, liveness, and startup probes, add one or more probes to the specification for the pod that contains the container which you want to perform the health checks
+To configure readiness and liveness probes, add one or more probes to the specification for the pod that contains the container which you want to perform the health checks
```
Comment (bobfuru) [inline modules/application-health-configuring.adoc:8]: s/which/that
also missing period

Also, this sentence is a bit unwieldy. I don't really understand "the pod that contains the container that you want to perform the health checks". Suggested rewrite to something like:

"To configure readiness and liveness probes, add one or more probes to the pod specification for the container that you want to perform the health checks on." (Not sure if that retains the meaning.)

**Review by bobfuru — COMMENTED** [review]
PR LGTM - a couple of suggested fixes elsewhere in one sentence.

**Comment by sunilcio** [comment]
@bergerhoffer hi, thanks for the edits. LGTM.

**Pull request merged**